pkg/services/ngalert/api/api_alertmanager.go

```diff
@@ -29,11 +26,10 @@ const (
 )
 
 type AlertmanagerSrv struct {
-	mam     *notifier.MultiOrgAlertmanager
-	secrets secrets.Service
-	store   AlertingStore
-	log     log.Logger
-	ac      accesscontrol.AccessControl
+	log        log.Logger
+	ac         accesscontrol.AccessControl
+	mam        *notifier.MultiOrgAlertmanager
+	encryption notifier.Crypto
```

Comment:
```suggestion
	crypto notifier.Crypto
```

nit: Same concept, more than decryption 😄

Reply:
Ah, thought I got all of them!